metricbeat/helper/server/http/http_test.go

```diff
@@ -212,6 +212,10 @@ func writeToServer(t *testing.T, message, host string, port int, connectionMetho
 	url := fmt.Sprintf("%s://%s:%d/", strings.ToLower(connectionType), host, port)
 	var str = []byte(message)
 	req, err := http.NewRequest(connectionMethod, url, bytes.NewBuffer(str))
+	if err != nil {
+		t.Error(err)
+		t.FailNow()
```

Comment:
That can be replaced by a single `t.Fatal()`
```suggestion
		t.Fatal(err)
```